rent_gpu/requestor/api/templates/offer-details.html

```diff
@@ -37,6 +37,136 @@ <h2>Provider status</h2>
+        <div class="panel">
+            <button class="accordion">Upload Dataset</button>
+            <div class="panel">
+              <code>
+                  scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -P {{ offer.port + 1 }} root@{{request.url.hostname}}:/usr/src/app/output/training/datasets/<b>[filename]</b> <b>[expected location/path on the user's computer]</b>
+              </code>
+            </div>
+            <button class="accordion">Upload Format</button>
+            <div class="panel">
+                <code>
+                scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -P {{ offer.port + 1 }} root@{{request.url.hostname}}:/usr/src/app/output/training/formats/<b>[filename]</b>  <b>[expected location/path on the user's computer]</b>
+                </code>
+            </div>
+            <button class="accordion">Upload Model</button>
+            <div class="panel">
+                  <code>
+                      scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -P {{ offer.port + 1 }} root@{{request.url.hostname}}:/usr/src/app/output/models/<b>[filename]</b> <b>[expected location/path on the user's computer]</b>
+                </code>
+            </div>
+            <button class="accordion">Upload LoRA</button>
+            <div class="panel">
+                <code>
+                scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -P {{ offer.port + 1 }} root@{{request.url.hostname}}:/usr/src/app/output/loras/<b>[filename]</b> <b>[expected location/path on the user's computer]</b>
+                </code>
+            </div>
+            <button class="accordion">Upload Prompt</button>
+            <div class="panel">
+                <code>
+                scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -P {{ offer.port + 1 }} root@{{request.url.hostname}}:/usr/src/app/output/prompts/<b>[filename]</b> <b>[expected location/path on the user's computer]</b>
+            </code>
+            </div>
+            <button class="accordion">Upload Preset</button>
+            <div class="panel">
+            <code>
+                  scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -P {{ offer.port + 1 }} root@{{request.url.hostname}}:/usr/src/app/output/presets/<b>[filename]</b> <b>[expected location/path on the user's computer]</b>
+            </code>
+            </div>
+        </div>
+
+        <script>
+            var acc = document.getElementsByClassName("accordion");
+            var i;
+
+            for (i = 0; i < acc.length; i++) {
+              acc[i].addEventListener("click", function() {
+                this.classList.toggle("active");
+                var panel = this.nextElementSibling;
+                if (panel.style.display === "block") {
+                  panel.style.display = "none";
+                } else {
+                  panel.style.display = "block";
```

Comment:
We can make it easier using toggle like on the other element.
```suggestion
                panel.classList.toggle("d-block");
```